autollm/utils/website_reader.py

```diff
@@ -8,13 +9,37 @@
 
 from autollm.utils.constants import WEBPAGE_READER_TIMEOUT
 from autollm.utils.logging import logger
-from autollm.utils.web_page_reader import WebPageReader
+from autollm.utils.webpage_reader import WebPageReader
 
 
-class WebDocsReader:
+class WebSiteReader:
 
-    def __init__(self):
+    def __init__(self, sitemap_url: Optional[str] = None):
```

Comment:
urlyi load fonksiyonunda vermelisin

```suggestion
    def __init__(self):
```